service/src/main/java/com/ge/predix/acs/monitoring/GraphDbHealthIndicator.java

```diff
@@ -0,0 +1,102 @@
+package com.ge.predix.acs.monitoring;
+
+import com.ge.predix.acs.model.Attribute;
+import com.ge.predix.acs.privilege.management.dao.GraphResourceRepository;
+import com.ge.predix.acs.privilege.management.dao.ResourceEntity;
+import com.ge.predix.acs.rest.Parent;
+import com.ge.predix.acs.zone.management.dao.ZoneEntity;
+import com.ge.predix.acs.zone.management.dao.ZoneRepository;
+import com.thinkaurelius.titan.core.QueryException;
+import com.thinkaurelius.titan.core.TitanConfigurationException;
+import com.thinkaurelius.titan.diskstorage.ResourceUnavailableException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.actuate.health.Health;
+import org.springframework.boot.actuate.health.HealthIndicator;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+
+import java.util.Collections;
+import java.util.Set;
+import java.util.UUID;
+
+@Component
+@Profile({ "titan" })
+public class GraphDbHealthIndicator implements HealthIndicator {
+
```

Comment:
Sure let's go with an `IN_MEMORY` status if `TITAN_ENABLE_CASSANDRA` is set to `false`